### ISSUE TITLE
Handling inheritence and related_names with interpolation

### ISSUE THREAD

**Issue by glassresistor** [django-extensions/django-extensions#1561]
Here is code for reference
```
class Bulkable(model.Model):
    """Uploadable data that can be modified
    """
    imported_from = models.ForeignKey(
        BulkImport,
        blank=True,
        null=True,
        related_name="%(app_label)s_%(class)s_set",
        on_delete=models.CASCADE,
    )

    class Meta:
        abstract = True

Class ABulkable(Bulkable):
    pass

Class BBulkable(Bulkable):
    pass
```
Arrow shows up as `imported_from (%(app_label)s_%(class)s_set)`, which makes sense since the Abstract model doesn't know that information yet.  Is it possible to make an option similar to -e/-E so that inheritance is handled by folding them in instead of having separate models shown?  

So instead of with ABulkable->Bulkable->BulkImport and BBulkable->Bulkable->BulkImport that ABulkable->BulkImport and all of the fields from Bulkable are included in A and B.  That would mean that related_name="%(app_label)s_%(class)s_set" could actually be interpolated and it would make it more consumable by non-programmers?

**Comment by trbs** [comment]
This is for `graph_models` ?

Happy to accept a PR for this :-)

**Comment by glassresistor** [comment]
Apologies for not referencing the command.  The code is a bit hard to follow but Ill see what I can do.
